parsers/s01-parse/hitech95/nginx-mail-logs.yaml

```diff
@@ -0,0 +1,51 @@
+onsuccess: next_stage
+name: crowdsecurity/nginx-mail-logs
+description: "Parse Nginx Mail logs"
```

Comment:
Hello it seems that the parser miss a filter. Can you add one please ?

Reply:
what you  mean with "missing a filter?"
There is no info in docs on what I should filter against:
https://docs.crowdsec.net/docs/parsers/create/

Reply:
docs is not clear on how the gating is done of the collections/single parsers etc.
I might have the solution but i'm not sure.